Generate.py

```diff
@@ -128,12 +128,12 @@ def main(args=None, callback=ERmain):
 
     if args.meta_file_path and os.path.exists(args.meta_file_path):
         try:
-            weights_cache[args.meta_file_path] = read_weights_yamls(args.meta_file_path)
+            meta_weights = read_weights_yamls(args.meta_file_path)[-1]
         except Exception as e:
             raise ValueError(f"File {args.meta_file_path} is destroyed. Please fix your yaml.") from e
-        meta_weights = weights_cache[args.meta_file_path][-1]
         print(f"Meta: {args.meta_file_path} >> {get_choice('meta_description', meta_weights)}")
-        del(meta_weights["meta_description"])
+        if "meta_description" in meta_weights:
```

Comment:
The old requirement for a meta_description had the side-effect of ensuring it is actually a meta file. By making it optional there is now no obvious error associated with that mistake.

Reply:
Is there a better way of verifying it or do you want me to revert this line?

Reply:
meta_description was kind of invented for that purpose. So unless you have a better solution it's probably best to bring it back and document the why this time.